api/BatteryManager.json

```diff
@@ -35,7 +35,7 @@
                   "name": "dom.battery.enabled"
                 }
               ],
-              "notes": "Disabled by default in Firefox 10, but can be enabled setting the preference <code>dom.battery.enabled</code> to <code>true</code>. Starting with Firefox 11, <code>mozBattery</code> is enabled by default. The Battery API is currently supported on Android, Windows, and Linux with UPower installed. Support for MacOS is available starting with Firefox 18. Firefox also provide support for the deprecated <a href='https://developer.mozilla.org/docs/Web/API/Navigator/battery' title='The battery read-only property returns a BatteryManager provides information about the system's battery charge level; provides also some new events you can handle to monitor the battery status. This implements the Battery Status API; see that documentation for additional details, a guide to using the API, and sample code.'><code>navigator.battery</code></a>."
+              "notes": "Disabled by default in Firefox 10, but can be enabled setting the preference <code>dom.battery.enabled</code> to <code>true</code>. Starting with Firefox 11, <code>mozBattery</code> is enabled by default. The Battery API is currently supported on Android, Windows, and Linux with UPower installed. Support for MacOS is available starting with Firefox 18. Firefox also provide support for the deprecated <a href='https://developer.mozilla.org/docs/Web/API/Navigator/battery' title=\"The battery read-only property returns a BatteryManager provides information about the system's battery charge level; provides also some new events you can handle to monitor the battery status. This implements the Battery Status API; see that documentation for additional details, a guide to using the API, and sample code.\"><code>navigator.battery</code></a>."
```

Comment:
Rather than change the quotes to escaped double-quotes, can we get rid of the `title` property entirely?  For our data, they have no real benefit -- we're also looking to lint against all attributes other than an `href` as it is.

Reply:
That will be removed in #4035.

Reply:
I think it's pointless to fix something just for it to be removed.  It would be great to just tackle those changes at once!

Reply:
Yeah, I would also just remove title attributes. They shouldn't have been there in the first place.

Reply:
Hey @ExE-Boss, do you plan on following up on this PR?